client/app/resources/collections-api.factory.js

```diff
@@ -5,7 +5,7 @@
     .factory('CollectionsApi', CollectionsApiFactory);
 
   /** @ngInject */
-  function CollectionsApiFactory($http, API_BASE) {
+  function CollectionsApiFactory($http, API_BASE, lodash) {
```

Comment:
Can we use lodash on node 0.10?

Reply:
Yep!  We already have throughout the code on the euwe branch